### PR TITLE
Added option to set the base CDN url as the root for absolute paths

### PR DESCRIPTION
**Pull request by kirkhoff**
https://github.com/callumlocke/grunt-cdnify/issues/18

tagging @callumlocke for review :)

### CODE REVIEW

**Comment by kirkhoff** [comment]
tagging @callumlocke for review :)

**Comment by XhmikosR** [comment]
@kirkhoff: please split the patches into separate PRs.

**Comment by XhmikosR** [comment]
And fetch and rebase.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "grunt-cdnify",
-  "version": "0.1.1",
+  "version": "0.1.2",
```
Comment (XhmikosR) [inline package.json]: Leave this out.

**Comment by XhmikosR** [comment]
@kirkhoff: actually, this PR's fallback patch should be relevant now. Regardless, it needs a rebase. If you can't do it ping me and I'll try to do it myself.

**Comment by XhmikosR** [comment]
So, @kirkhoff will you clean this up or we should just close it?

**Comment by kirkhoff** [comment]
@XhmikosR I'll rebase and send up an update today. Thanks :)

**Comment by kirkhoff** [comment]
Do you still want me to remove the patch version change in package.json?

**Comment by XhmikosR** [comment]
Yes, it's up to the maintainer(s).

**Comment by XhmikosR** [comment]
So, do you plan to rebase this or not @kirkhoff?

**Comment by kirkhoff** [comment]
@XhmikosR with the new `url` dependency, the code is entirely different. Closing the PR. If the issue still exists, I can refactor my change and send up a new pr later.

**Comment by XhmikosR** [comment]
No idea what the issue you had was so just check yourself.

**Pull request closed without merging**